platform/lib/templates/index.js

```diff
@@ -27,11 +27,32 @@ const {pagePath, paths} = require('../utils/project');
 
 let templates = null;
 
+/**
+ * Builds a Object that adds mandatory variables to the render context
+ * that are needed to successfully SSR the pages
+ * @param  {expressjs.Request} request
+ * @return {Object}
+ */
+function context(request, base) {
+  request = request || {'query': {}};
+  const context = {} || base;
+
+  const ALLOWED_FORMATS = ['websites', 'stories', 'ads', 'email'];
```

Comment:
Templates.js feels like the wrong place for this. Can we move this to the caller?

Reply:
My reasoning was that potentially any template that is loaded here could rely on having `format` defined. If we move it to the caller we'd need to define it in both: 

https://github.com/ampproject/docs/blob/7c944d4f14e44d8e74282e117a086ed96931edea/examples/source/1.components/amp-live-list/api.js#L22

and 

https://github.com/ampproject/docs/blob/7c944d4f14e44d8e74282e117a086ed96931edea/platform/lib/routers/pages.js#L162